### PR TITLE
Migrate breakpoint names for font-size utility classes 

### PR DESCRIPTION
**Pull request by jamigibbs**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

This PR will migrate the breakpoint names for the [font-size utility classes](https://design.va.gov/foundation/utilities/font-size).

| Current VADS Name | USWDS v3 Name |
| --------------------------- | ------------------------- |
| xsmall-screen (320px)          | mobile (320px) |
| small-screen (480px) | mobile-lg (480px) |
| small-desktop-screen (1008px) | desktop (1024px) 
| large-screen (1200px) | desktop-lg (1200px) |

## Related issue(s)

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3209

## Testing done

- confirmed locally that the new breakpoint names were applying from the CSS Library utilities stylesheet (screenshots inline)

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/accredited-representative-portal/components/LandingPageWelcome/LandingPageWelcome.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const LandingPageWelcome = ({ firstName = '', children }) => (
               Welcome to the Accredited Representative Portal
               {firstName && `, ${firstName}`}
             </h1>
-            <h2 className="vads-u-color--white vads-u-margin-top--3 vads-u-font-size--xl small-desktop-screen:vads-u-font-size--2xl">
+            <h2 className="vads-u-color--white vads-u-margin-top--3 vads-u-font-size--xl desktop:vads-u-font-size--2xl">
```
Comment (jamigibbs) [inline src/applications/accredited-representative-portal/components/LandingPageWelcome/LandingPageWelcome.jsx:22]: Screenshot 2024-09-26 at 10 22 44 AM

File: src/applications/gi/components/profile/ContactInformation.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default function ContactInformation({ institution, showModal }) {
     institution.physicalAddress1 && (
       <div className="vads-l-row vads-u-margin-top--2p5 vads-u-margin-bottom--4">
         <div className="vads-l-col--12 medium-screen:vads-l-col--3">
-          <h3 className="small-screen:vads-u-font-size--h4 contact-heading vads-u-font-family--sans vads-u-margin--0 small-screen-font">
+          <h3 className="mobile-lg:vads-u-font-size--h4 contact-heading vads-u-font-family--sans vads-u-margin--0 small-screen-font">
```
Comment (jamigibbs) [inline src/applications/gi/components/profile/ContactInformation.jsx:26]: Screenshot 2024-09-26 at 10 28 17 AM

File: src/applications/gi/components/profile/ContactInformation.jsx [hunk 3]
```diff
@@ -51,7 +51,7 @@ export default function ContactInformation({ institution, showModal }) {
     institution.address1 && (
       <div className="vads-l-row vads-u-margin-top--0 vads-u-margin-bottom--4">
         <div className="vads-l-col--12 medium-screen:vads-l-col--3">
-          <h3 className="small-screen:vads-u-font-size--h4 contact-heading vads-u-font-family--sans vads-u-margin--0 small-screen-font">
+          <h3 className="mobile-lg:vads-u-font-size--h4 contact-heading vads-u-font-family--sans vads-u-margin--0 small-screen-font">
```
Comment (jamigibbs) [inline src/applications/gi/components/profile/ContactInformation.jsx:54]: Screenshot 2024-09-26 at 10 22 56 AM

File: src/applications/gi/components/profile/schoolRatings/RatingHeading.jsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ export const RatingHeading = ({ ratingCount, displayStars, ratingAverage }) => {
         </div>
 
         <div className="vads-l-row">
-          <div className="vads-l-col vads-u-font-weight--bold small-screen:vads-u-font-size--base vads-u-font-family--serif small-screen-font">
+          <div className="vads-l-col vads-u-font-weight--bold mobile-lg:vads-u-font-size--base vads-u-font-family--serif small-screen-font">
```
Comment (jamigibbs) [inline src/applications/gi/components/profile/schoolRatings/RatingHeading.jsx:17]: Screenshot 2024-09-26 at 10 23 03 AM

File: src/applications/gi/components/profile/schoolRatings/RatingsAbout.jsx [hunk 5]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const RatingsAbout = () => {
   return (
     <div className="vads-u-padding-top--4 about-ratings">
-      <div className="small-screen:vads-u-font-size--h3 vads-u-padding-bottom--1p5 vads-u-font-weight--bold vads-u-font-family--serif small-screen-font">
+      <div className="mobile-lg:vads-u-font-size--h3 vads-u-padding-bottom--1p5 vads-u-font-weight--bold vads-u-font-family--serif small-screen-font">
```
Comment (jamigibbs) [inline src/applications/gi/components/profile/schoolRatings/RatingsAbout.jsx:6]: Screenshot 2024-09-26 at 10 23 09 AM

File: src/applications/gi/components/profile/schoolRatings/RatingsAbout.jsx [hunk 6]
```diff
@@ -12,7 +12,7 @@ const RatingsAbout = () => {
         they’ve attended on a scale of 1 to 4 stars, with 4 stars being the best
         rating.
       </p>
-      <span className="small-screen:vads-u-font-size--h4 vads-u-font-weight--bold small-screen-font">
+      <span className="mobile-lg:vads-u-font-size--h4 vads-u-font-weight--bold small-screen-font">
```
Comment (jamigibbs) [inline src/applications/gi/components/profile/schoolRatings/RatingsAbout.jsx:15]: Screenshot 2024-09-26 at 10 23 17 AM

File: src/applications/static-pages/facilities/FacilityPatientSatisfactionScoresWidget.jsx [hunk 7]
```diff
@@ -28,7 +28,7 @@ export function FacilityPatientSatisfactionScoresWidget(props) {
     <div>
       <h2
         id="our-patient-satisfaction-scores"
-        className="vads-u-margin-top--4 vads-u-font-size--lg small-screen:vads-u-font-size--xl"
+        className="vads-u-margin-top--4 vads-u-font-size--lg mobile-lg:vads-u-font-size--xl"
```
Comment (jamigibbs) [inline src/applications/static-pages/facilities/FacilityPatientSatisfactionScoresWidget.jsx:31]: Screenshot 2024-09-26 at 10 23 24 AM

File: src/applications/vaos/components/calendar/CalendarNavigation.jsx [hunk 8]
```diff
@@ -43,7 +43,7 @@ const CalendarNavigation = ({
       <span role="cell">
         <h2
           id={`h2-${momentMonth.format('YYYY-MM')}`}
-          className="vads-u-font-size--h4 small-screen:vads-u-font-size--h3 vads-u-margin-top--0p5 vads-u-font-weight--bold vads-u-text-align--center vads-u-align-items--center vads-u-margin-bottom--0 vads-u-display--block vads-u-font-family--serif"
+          className="vads-u-font-size--h4 mobile-lg:vads-u-font-size--h3 vads-u-margin-top--0p5 vads-u-font-weight--bold vads-u-text-align--center vads-u-align-items--center vads-u-margin-bottom--0 vads-u-display--block vads-u-font-family--serif"
```
Comment (jamigibbs) [inline src/applications/vaos/components/calendar/CalendarNavigation.jsx:46]: Screenshot 2024-09-26 at 10 23 30 AM

**Review by tjheffner — APPROVED** [review]
changes under `accredited-representative-portal/` LGTM

**Review by Andrew565 — APPROVED** [review]

**Review by eselkin — COMMENTED** [review]
LGTM

**Pull request merged**